### ISSUE TITLE
Themes

### ISSUE THREAD

**Issue by KentEkl** [tomasrudh/analogclock#3]
Hej Tomas.
How do I do to change the color of the background, I use the theme midnight for my display but the square around the clock stays white.

title: iPad
views:
  - path: default_view
    title: Home
    theme: midnight
    panel: false
    badges: []
    cards:

**Comment by tomasrudh** [comment]
Hej,

The card background is set by the property 'card-background-color' in themes. Try to set that, and see if that works for you.

Thanks for reporting!
/Tomas

**Comment by KentEkl** [comment]
I'm not sure if I'm following you.
I'm using the default for theme Backend-selected for Home assistant on iPad ie. the normal for Hassio, but then
I have theme: midnight for the view in this Lovelace dashboard as you can see in the supplied code and that changes everything exept the background for the analog clock to dark grey.

title: iPad
views:
  - path: default_view
    title: Home
    _**theme: midnight**_
    panel: false
    badges: []
    cards:
      - cards:
          - entity: >-
              sensor.desktop_g03k04t_intel_core_i5_8259u_temperatures_cpu_package
            scale: 29px
            sensor: null
            title: NUC CPU Temp
            type: 'custom:bignumber-card'
          - entity: sensor.desktop_g03k04t_intel_core_i5_8259u_load_cpu_total
            scale: 29px
            sensor: null
            title: NUC CPU %
            type: 'custom:bignumber-card'
          - entity: sensor.desktop_g03k04t_generic_memory_data_used_memory
            scale: 29px
            sensor: null
            title: NUC Memory GB
            type: 'custom:bignumber-card'
        type: vertical-stack
      - cards:
          - cards:
              - entity: sensor.date
                type: 'custom:bignumber-card'
              - entity: sensor.netatmo_dalagatan_temperature
                scale: 45px
                sensor: null
                title: Inomhus
                type: 'custom:bignumber-card'
            type: vertical-stack
          - type: media-control
            entity: media_player.beam
          - type: horizontal-stack
            cards:
              - type: 'custom:button-card'
                entity: switch.sovrum_a
                color_type: card
                icon: 'mdi:lamp'
              - type: 'custom:button-card'
                entity: switch.sovrum_b
                color_type: card
                icon: 'mdi:lamp'
              - type: 'custom:button-card'
                entity: switch.kok
                color_type: card
                icon: 'mdi:lamp'
              - type: 'custom:button-card'
                entity: switch.fibaro_system_fgwpe_f_wall_plug_gen5_switch_2
                color_type: card
                icon: 'mdi:fan'
                name: Fläkt
        type: vertical-stack
      - type: 'custom:analog-clock'
        hide_SecondHand: true
        locale: sv-SE
        diameter: 300
        color_Background: '#454545'
        color_HourHand: '#FFFFFF'
        color_MinuteHand: '#FFFFFF'
        color_DigitalTime: '#FFFFFF'
        color_FaceDigits: '#FFFFFF'
        color_Ticks: '#FFFFFF'
        color_Text: '#FFFFFF'

**Comment by tomasrudh** [comment]
There is a file in the HA root called themes.yaml, that contains all the colors for Lovelace, separated into different themes.

There seems to be a problem with the card background setting since some version of HA (0.112?), I would think the setting 'paper-card-background-color' would set the background color for the cards, but it doesn't. Instead, the setting 'card-background-color' do set the correct color. When I apply midnight do all cards get a white background, except those that has a picture as background. Also the theme 'default' set all of my cards backgrounds to white.

In the file themes.yaml add this line:
card-background-color: 'var(--paper-card-background-color)'

/Tomas

**Comment by tomasrudh** [comment]
Your themes might also be configured in configuration.yaml instead of themes.yaml.

**Comment by KentEkl** [comment]
Hej igen.
I found out that something has changed since I installed the themes some years ago.
I reinstalled the themes and now HACS organizes the themes in a directory structure named themes with sub directories.
'config/themes/midnight/midnight.yaml' since there is a new subdiretory for each new theme I had to include them with
 'themes: !include_dir_merge_named themes' in configuration.yaml , after that everything works fine without changing anything else. The background is grey as I wanted.
Tackar ödmjukast för din tid. :-)
